scripts/deploy.sh

```diff
@@ -35,6 +35,7 @@ gcloud --quiet auth configure-docker
 
 
 echo "Generating keys..."
+rm -f ./genfiles/*
```

Comment:
Before running a command like this, it seems prudent to check/enforce the current directory is what you expect it to be.